### PR TITLE
Allow opening the staged diff in a vsplit

### LINKED ISSUE #1293: Allow `show_staged_diff` to open in a vertical split
**Is your feature request related to a problem? Please describe.**
No.

**Describe the solution you'd like**
I'd like to be able to configure `show_staged_diff` to open in a vertical split. At the moment it always opens the diff buffer at the bottom, but I'd love to see it to the right.

**Describe alternatives you've considered**

**Additional context**
I'd love to send you a PR if you help me by pointing out where the code should go.

### PR DESCRIPTION
**Pull request by ptn**
Fixes #1293

### CODE REVIEW

File: lua/neogit/config.lua [hunk 1]
```diff
@@ -69,9 +69,14 @@ end
 ---@class NeogitConfigPopup Popup window options
 ---@field kind WindowKind The type of window that should be opened
 
+---@alias StagedDiffSplitKind
+---| "split" Open in a split
+---| "vsplit" Open in a vertical split
```
Comment (CKolkey) [inline lua/neogit/config.lua:74]: This can also include `split_above` and `auto`. `tab` and `floating` would _work_, but I think the result would be poor.

**Comment by CKolkey** [comment]
Thanks! Small addition to the type annotation, but otherwise looks good :)

File: lua/neogit/config.lua (hunk 1 above)
Comment (ptn) [reply lua/neogit/config.lua:74]: Makes sense. Pushed.

Comment (CKolkey) [reply lua/neogit/config.lua:74]: Thanks!

**Pull request merged**